domains/Blog/components/TagSelect/TagSelect.js

```diff
@@ -0,0 +1,56 @@
+import PropTypes from 'prop-types'
+import Slider from 'react-slick'
+import { Text } from '../../../../components'
+
+const TagSelect = (props) => {
+  const { title, onChange, tags, carousel } = props
+
+  const settings = {
+    dots: false,
+    infinite: false,
+    slidesToScroll: 1,
+    variableWidth: true
+  }
+
+  return (
+    <div className="row">
+      {title && (
+        <div className="col-12">
+          <p className="subtitle">{title}</p>
```

Comment:
you use <p> and <Text>, better to use component Text and pass an appropriate prop (subtitle, etc)